examples/mecci_kissinger.py

```
@@ -0,0 +1,234 @@
+
+    # User specifies peak type
+    peak_type = input("Is the peak exothermic or endothermic? (Enter 'exo' or 'endo'): ").lower()
+    exothermic = peak_type == 'exo'
+
+    # Plot the curve again with the selected range highlighted
+    plot_dsc_with_derivatives(temp, hf, sample_file, temp_range, exothermic=exothermic)
+
+    # Allow user to adjust peak detection parameters
+    adjust = input("Do you want to adjust peak detection parameters? (y/n): ").lower()
+    if adjust == 'y':
+        prominence = float(input("Enter minimum peak prominence (default is 0.1): ") or 0.1)
```

Ruff

Local variable `prominence` is assigned to but never used

Remove assignment to unused variable `prominence`

(F841)